app/services/discourse_service.rb

```diff
@@ -1,6 +1,14 @@
 class DiscourseService
   attr_reader :user, :client
 
+  # This is the map to match custom_fields with Discourse user fields
+  USER_FIELDS_MAP = {
```

Comment:
So now we map custom user fields with the data we already have, right?

Reply:
yeah, this table is to make it easier match our data with Discourse. You can only pass the field `id` instead of `name` in the payload.